spec/support/vcr.rb

```diff
@@ -87,7 +87,7 @@ def scrub_data(data, scrubber)
         data.each_value do |top_values|
           case top_values
           when Array
-            top_values.each_key do |nested_item|
```

Comment:
question: why the change and what does it do?

Reply:
At the time it was a linting issue causing the cassettes to not be scrubbed correctly. I can't seem to recreate that error anymore, so I'll revert this.